rainbow-me/rainbow
# Remove uniswap pairs redux

Fixes APP-821

## What changed (plus any additional context for devs)
Removed uniswap pairs redux. Why does this need to be stored in state at all? `rainbowTokenList` has state of its own that receives periodic updates. Instead of storing + accessing this data via redux state, we now directly access it from `rainbowTokenList`.

## Screen recordings / screenshots

https://github.com/rainbow-me/rainbow/assets/15272675/0420b408-3645-4141-927b-ae77e4012aa6


## What to test
1. Make sure the above logic is sound
2. Check that tokens are present in swap currency select modal + discover search
